requirements-test.txt

```diff
@@ -6,5 +6,5 @@ nose>=1.3.0
 django-nose>=1.2
 flake8>=2.1.0
 tox>=1.7.0
```

Comment:
I see we're including Tox, but we're not really using it. I suggest we fully embrace Tox!